### ISSUE TITLE
PySide6/__init__.py: Unable to import Shiboken from

### ISSUE THREAD

**Issue by icefairy** [Nuitka/Nuitka#1786]
python -m nuitka --version
1.0.7
Commercial: None
Python: 3.7.13 (default, Mar 28 2022, 08:03:21) [MSC v.1916 64 bit (AMD64)]
Flavor: Anaconda Python
Executable: d:\soft\conda\envs\icejr\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10

 * How did you install Nuitka and Python
pip install nuitka 
with gcc：
gcc (MinGW-W64 x86_64-posix-seh, built by Brecht Sanders) 11.3.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

  > python -m pip freeze
`altgraph==0.17.2
appdirs==1.4.4
attrs==22.1.0
cache3==0.3.1
cattrs==22.1.0
certifi @ file:///C:/Windows/TEMP/abs_e9b7158a-aa56-4a5b-87b6-c00d295b01fanefpc8_o/croots/recipe/certifi_1655968940823/work/certifi
charset-normalizer==2.1.1
colorama==0.4.5
commonmark==0.9.1
decorator==5.1.1
efinance==0.4.9
exceptiongroup==1.0.0rc9
future==0.18.2
greenlet==1.1.3
idna==3.3
importlib-metadata==4.12.0
jsonpath==0.82
loguru==0.6.0
multitasking==0.0.11
numpy==1.21.6
pandas==1.3.5
pefile==2022.5.30
psycopg2==2.9.3
py==1.11.0
pyaml==21.10.1
Pygments==2.13.0
pyinstaller==5.3
pyinstaller-hooks-contrib==2022.9
PySide6==6.3.1
PySide6-Addons==6.3.1
PySide6-Essentials==6.3.1
python-dateutil==2.8.2
pytz==2022.2.1
pywin32-ctypes==0.2.0
PyYAML==6.0
requests==2.28.1
requests-cache==0.9.6
retry==0.9.2
rich==12.5.1
shiboken6==6.3.1
six==1.16.0
SQLAlchemy==1.4.40
TA-Lib @ file:///D:/down/TA_Lib-0.4.24-cp37-cp37m-win_amd64.whl
tqdm==4.64.0
typing_extensions==4.3.0
url-normalize==1.4.3
urllib3==1.26.12
win32-setctime==1.1.0
wincertstore==0.2
zipp==3.8.1
`

* Provide in your issue the Nuitka options used
`
nuitka --standalone --show-progress  --nofollow-imports --show-memory --enable-plugin=pyside6,numpy,upx --include-data-files="config.yaml=config.yaml" --follow-import-to=src   --output-dir=dist main.py
`
compile and package success ，but error with run：
`
PySide6/__init__.py: Unable to import Shiboken from D:\codes\THREE-~1\dist\main.dist, D:\codes\THREE-~1\dist\MAIN~1.DIS
Traceback (most recent call last):
  File "D:\codes\THREE-~1\dist\MAIN~1.DIS\main.py", line 1, in <module>
  File "D:\codes\THREE-~1\dist\MAIN~1.DIS\PySide6\__init__.py", line 144, in <module PySide6>
  File "D:\codes\THREE-~1\dist\MAIN~1.DIS\PySide6\__init__.py", line 64, in _setupQtDirectories
  File "D:\codes\THREE-~1\dist\MAIN~1.DIS\shiboken6\__init__.py", line 26, in <module shiboken6>
ImportError
`
there was shiboken6/Shiboken.pyd in the dist directory.

**Comment by kayhayen** [comment]
Asking Nuitka to no follow imports and then being suprised then it cannot import something, well, this is a duplicate of wide spread user error that I cannot understand in the light of the red warning it gives already.

**Comment by kayhayen** [comment]
These is already a report about things.

**Comment by icefairy** [comment]
> Asking Nuitka to no follow imports and then being suprised then it cannot import something, well, this is a duplicate of wide spread user error that I cannot understand in the light of the red warning it gives already.

sorry, but the same error when I run it with out "--nofollow-imports" param
